packages/office-desk-esp32c3.yaml

```diff
@@ -0,0 +1,275 @@
+substitutions:
+  device_name: Flexispot EK5
+  name: office-desk-flexispot-ek5
+  min_height: "73.5" # cm
+  max_height: "123" # cm
+  ssid: "your_wifi_ssid"
+  wifi_password: "your_wifi_password"
+  ap_fallback_password: "MnANX95MWad1"
+  tx_pin: GPIO21 # TX
+  rx_pin: GPIO20 # RX
+  screen_pin: GPIO10
+  encryption_key: "iOZqtvw31Yy6sasRl5h2DElG2VDlqW2WjJEKObVN8bg="
+
+external_components:
+  source: github://iMicknl/LoctekMotion_IoT
+  components: [loctekmotion_desk_height]
+
+esp32:
```

Comment:
This is still the esp32 config

Reply:
This should be the same for `esp32c3`, only the `variant:` might be different.

Personally I don't think it makes sense to add another configuration, for such minor changes. Perhaps we can make the README more clear on what to change for ESP8266 and ESP32 variations.

Reply:
I think I agree. Multiple example configs means more maintenance.